docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeDockerConfiguration.java

```diff
@@ -307,4 +315,24 @@ private static String resolveFilename(URI uri) {
 
         return content.toString();
     }
+
+    public static String resolveUrl(String machineVersion) {
+        validation(machineVersion, DOCKER_MACHINE_VERSION);
```

Comment:
At this time we could use *curl -s https://api.github.com/repos/docker/machine/releases/latest* in order to get the **name** property. For example, *v0.5.6*. Then use the name's value in **machineVersion**.